sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProto.java

```diff
@@ -479,15 +502,41 @@ public static DynamicMessage messageFromTableRow(
         }
       }
 
+      if (unknownFields != null) {
```

Comment:
Is this file the conversion stage? 

If user has input ABC and the original schema is AB but later on the schema is updated to ABC, will L496 fail the check?

Reply:
no, because the caller (StorageApiDynamicDestinationsTableRow.java:151) passes in true if autoSchemaUpdates==true